service/src/integrationTest/kotlin/fi/espoo/evaka/application/ApplicationControllerCitizenIntegrationTest.kt

```diff
@@ -0,0 +1,141 @@
+// SPDX-FileCopyrightText: 2017-2023 City of Espoo
+//
+// SPDX-License-Identifier: LGPL-2.1-or-later
+
+package fi.espoo.evaka.application
+
+import fi.espoo.evaka.FullApplicationTest
+import fi.espoo.evaka.insertApplication as insertTestApplication
+import fi.espoo.evaka.placement.PlacementType
+import fi.espoo.evaka.shared.auth.AuthenticatedUser
+import fi.espoo.evaka.shared.auth.CitizenAuthLevel
+import fi.espoo.evaka.shared.auth.UserRole
+import fi.espoo.evaka.shared.dev.insertTestCareArea
+import fi.espoo.evaka.shared.dev.insertTestDaycare
+import fi.espoo.evaka.shared.dev.insertTestEmployee
+import fi.espoo.evaka.shared.dev.insertTestPerson
+import fi.espoo.evaka.shared.domain.BadRequest
+import fi.espoo.evaka.shared.domain.HelsinkiDateTime
+import fi.espoo.evaka.shared.domain.MockEvakaClock
+import fi.espoo.evaka.test.getApplicationStatus
+import fi.espoo.evaka.testAdult_1
+import fi.espoo.evaka.testArea
+import fi.espoo.evaka.testChild_1
+import fi.espoo.evaka.testDaycare
+import fi.espoo.evaka.testDecisionMaker_1
+import kotlin.test.Test
+import kotlin.test.assertEquals
+import kotlin.test.assertNull
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.assertThrows
+import org.springframework.beans.factory.annotation.Autowired
+
+class ApplicationControllerCitizenIntegrationTest : FullApplicationTest(resetDbBeforeEach = true) {
+    @Autowired private lateinit var applicationControllerCitizen: ApplicationControllerCitizen
+
+    @Autowired private lateinit var stateService: ApplicationStateService
+
+    private val clock = MockEvakaClock(HelsinkiDateTime.now())
```

Comment:
Mock clock + now is not a very useful combo, because the test still depends on the time it's executed. If the precise timestamp is not important in the test, just pick one timestamp and hard-code it here.